### ISSUE TITLE
Application testing

### ISSUE THREAD

**Issue by MartaSien** [zetonteam/zeton_django#83]
- [ ] Define necessary test coverage
- [ ] Implement tests
- [x] Test automation?

### Related 
- https://github.com/zetonteam/zeton_django/pull/67

**Comment by bartmkraus** [comment]
I'd be happy to write some tests. It would be nice if one day we could discuss strategies, goals, scenarios and exchange resources on this topic. I think if we keep the coverage at 80-90%, we'll be fine.

**Comment by pyexplorerhs3** [comment]
Current tests are not in use, e.g https://app.circleci.com/pipelines/github/zetonteam/zeton_django/255/workflows/cd6af1f3-05d3-4518-9216-0b659b019ba4/jobs/259/steps

```
Found 0 test(s).
System check identified no issues (0 silenced).

----------------------------------------------------------------------
Ran 0 tests in 0.000s

NO TESTS RAN
```

**Comment by MartaSien** [comment]
Unit tests are in implementation. CircleCI config is currently running unit tests, on commits and PRs. It has been implemented in #149

**Comment by arkjedrz** [comment]
@MartaSien 
I guess this topic can be closed. There are tickets for remaining endpoints, and coverage is not a priority right now.
